config/default.config

```diff
@@ -18,7 +18,6 @@ params {
 
     //algorithm version
     pipeval_version = "4.0.0-rc.2"
-    samtools_version = "1.16.1"
```

Comment:
The only samtools still being used in the pipeline is included within the SomaticSniper package.